src/main.cc

```diff
@@ -1174,6 +1174,8 @@ mainInitialize(void)
 
 #endif
 
+    // TODO make these init() calls happen as RegisteredRunner::finalizeConfig
```

Comment:
The exact scope of the "these init() calls" phrase is unclear, and not all of the initialization calls below should be converted to finalizeConfig. I suggest dropping this out-of-scope change to avoid arguing about it.

Reply:
Done in c6b6dbd